src/components/dls/QuranWord/TextWord.tsx

```diff
@@ -10,6 +10,7 @@ import { CharType } from 'types/Word';
 type MadaniWordTextProps = {
   text: string;
   font: string;
+  cssClass?: string;
```

Comment:
1. `cssClass` should be renamed to `className` for consistency for example see `PopoverMenuItemProps` or `ButtonProps`.
2. I cannot think of why we need this prop because we can add the styles directly inside `styles.word` since `TextWord` component is only used once and by only `QuranWord` component